src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -2456,16 +2488,16 @@ private bool HasVariantConversion(NamedTypeSymbol source, NamedTypeSymbol destin
                 return quickResult.Value();
             }
 
-            return this.CreateInstance(_currentRecursionDepth + 1).
+            return this.CreateInstance(_currentRecursionDepth + 1, _withoutNullability).
```

Comment:
The `_withoutNullability` instance did not increment its recursion depth. Does that matter?
May not be relevant if `_withoutNullability` can be removed in favor of a boolean flag.